pkg/action/build-iso.go

```diff
@@ -183,6 +184,27 @@ func (b BuildISOAction) createEFI(rootdir string, isoDir string) error {
 		b.cfg.Logger.Errorf("Failed writing grub.cfg: %v", err)
 		return err
 	}
+	// Ubuntu efi searches for the grub.cfg file under /EFI/ubuntu/grub.cfg while we store it under /boot/grub2/grub.cfg
+	// workaround this by copying it there as well
+	// read the os-release from the rootfs to know if we are creating a ubuntu based iso
+	FLAVOR, err := sdk.OSRelease("FLAVOR", filepath.Join(rootdir, "etc/os-release"))
```

Comment:
[Minor] No reason to use upper cased variable names. Just `flavor` should be ok.

Reply:
```
ERRO[2024-01-12T14:22:07Z] Failed preparing ISO's root tree: KAIROS_flavor key not found 
```

case sensitive!

Reply:
:facepalm:

Reply:
sorry yes, I meant the golang variable, not the `os-release` one.